do.sh

```diff
@@ -322,11 +280,21 @@ function init_ovn_fake_multinode() {
 
 # Pull image on all nodes
 function pull_ovn_fake_multinode() {
-    # Pull image on all nodes
+    pushd ${rundir}/ovn-fake-multinode
+
+    # Save the image locally.
+    rm -f ovn-multi-node-image.tar
+    podman save -o ovn-multi-node-image.tar ovn/ovn-multi-node:latest
     ansible-playbook ${ovn_fmn_playbooks}/pull-fake-multinode.yml -i ${hosts_file}
+    popd
 }
 
 function pull_ovn_tester() {
+    # Save the image locally, use docker-dir format because it allows
+    # compression.
+    rm -f ovn-tester-image.tar
+    podman save -o ovn-tester-image.tar ovn/ovn-tester:latest
+
     ansible-playbook ${ovn_fmn_playbooks}/pull-ovn-tester.yml -i ${hosts_file}
 }
```

Comment:
Could you add an echo saying what is happening before each  `podman save`?  It would be easier to understand what is going on.

Also the comment in `pull_ovn_tester()` is not correct as we're not using `docker-dir`.
Speaking of formats though, can we use oci-archive instead?  It provides compression for layers by default and is producing 3x smaller image:
```
-rw-r--r-- 1 root root 1.2G Jun  1 18:10 /tmp/docker-archive.tar
-rw-r--r-- 1 root root 468M Jun  1 18:09 /tmp/oci-archive.tar
```
Above are results for an actual `ovn-multi-node` image in different formats.  The size difference also explained here: https://github.com/containers/podman/discussions/17934
It takes a noticeable amount of time to sync 1.2 GB between many physical hosts.

Reply:
Cool, I didn't know about oci-archive.  I'll use that and fix all the other issues too, thanks!

Reply:
Done.